src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsMongoDbRepository.cs

```diff
@@ -17,7 +16,15 @@ public class ProjectsMongoDbRepository : IProjectsMongoDbRepository
         public ProjectsMongoDbRepository()
         {
             string stringConnection = Environment.GetEnvironmentVariable("MongoSessionServices", EnvironmentVariableTarget.Process);
-            client = new MongoClient(stringConnection);
+            try
+            {
+                client = new MongoClient(stringConnection);
+            }
+            catch (Exception)
+            {
+                throw;
```

Comment:
you should manage try catch in core layer

Reply:
This was removed